Gemfile

```diff
@@ -20,6 +20,7 @@ gem "gds-sso", "12.1.0"
 gem 'bunny', '2.5.1'
 gem 'whenever', '0.9.4', require: false
 gem "govuk_sidekiq", "~> 0.0"
+gem 'sidekiq', git: "https://github.com/alphagov/sidekiq.git", branch: "fix-testing-with-overridden-queue"
```

Comment:
No sorry, I mean why is the branch not merged into master? It also looks like it is behind? 
`This branch is 1 commit ahead, 81 commits behind mperham:master.`
Also what was the reason for not fixing this in mperham/sidekiq?